src/globals.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
any particular reason to pull this out into its own file? Is it to remove the build time step needed for our own linting to work? If so, please open up a separate PR with it (love the idea, though!)

Reply:
It was b/c `.eslintrc.js` can't read typescript files, so I had to break it out to be able to import it into for linting while editing.

Reply:
I've made #366